docs/admin.rst

```diff
@@ -42,3 +42,52 @@ to your MPTTModelAdmin::
     class CustomMPTTModelAdmin(MPTTModelAdmin):
         mptt_indent_field = "some_node_field"
     # …
+
+
+``mptt.admin.DraggableMPTTAdmin``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.9
+
+This is a tree admin based on `FeinCMS <http://feincms.org/>`_ offering
+drag-drop functionality for moving nodes.
+
+.. warning::
+
+   Does not work well with big trees (more than a few hundred nodes, or trees
+   deeper than 10 levels). Patches implementing lazy-loading of deep trees
+   are very much appreciated.
+
+``DraggableMPTTAdmin.list_per_page`` is set to 2000 by default (which
+effectively disables pagination for most trees).
+
+Usage::
+
+    from django.contrib import admin
+    from mptt.admin import DraggableMPTTAdmin
+    from myproject.myapp.models import Node
+
+    admin.site.register(
+        Node,
+        DraggableMPTTAdmin,
+        list_display=('__str__', ...more fields if you feel like it...),
+    )
+
+Two columns are inserted after the action checkbox column with tree actions
+(move node, toggle subtrees) and an indented title according to the level of the
+tree node. The indented title takes its value from the model instance's
+``__str__`` method. The ``__str__`` entry itself is automatically removed
+from ``list_display``, but it is still recommended to leave it in there for
+clarity.
+
```

Comment:
One thing that'd be useful to add here: what browsers does the javascript work on? I see a compat fix for IE<9 down there, does that mean it works in IE8?